great-idea/css/index.css

```diff
@@ -63,4 +63,168 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+
+header {
+	height: 70px;
+}
+
+nav{
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+a {
+	margin: 0 10px;
+  	padding: 20px;
+  	border-radius: 20px;
+  	color: gray;
+	margin-bottom: 20px;
+	text-decoration: none;
+	
+}	
+
+.logo{
+	position: absolute;
```

Comment:
Try not to use absolute positioning unless you really have to because it will be harder to align everything else with this element. Use flexbox - ` justify-content:center; align-items:center ` BOOM. you are done